_includes/episode_keypoints.html

```diff
@@ -1,10 +1,17 @@
 {% comment %}
   Display key points for an episode.
 {% endcomment %}
+
+{% if page.keypoints == null %}
```

Comment:
I'm a tad confused here as "nothing" in Liquid is `nil`, not `null`. Also, we could check whether we have something in `page.keypoints` with `{%- if page.keypoints -%}` and avoid having to check `== null` completely. The only case where`page.keypoints == nil` is if `page.keypoints` can be an empty string, which evaluates to "true" in Liquid.